jbmc/regression/jbmc/temp_stack_variable_packing/test.desc

```diff
@@ -0,0 +1,6 @@
+CORE
+Primary.class
+--function stuff.array_issue.Primary.Run
+^VERIFICATION SUCCESSFUL|VERIFICATION FAILED$
```

Comment:
We really don't know which one to expect?

Reply:
We don't really mind, as long as it completes in some form the fix will still be in place (it just exceptioned half way through before).

Reply:
True, but generally we should know what our tests are doing and specify that. In this case I recommend putting an `assert(false)` in the `Accept` method and checking that we have `VERIFICATION FAILED` and also that we failed that particular assertion, thus establishing that the test explored the path where we make the call (cf. failing an earlier null-pointer check or something) as we intended.